tests/test_memory_core/build_tb.py

```diff
@@ -2140,6 +2140,33 @@ def software_gold(app_name, matrix_tmp_dir, give_tensor=False, print_inputs=None
         output_matrix = numpy.maximum(0, (numpy.matmul(b_mat, c_mat_trans, dtype=numpy.int16, casting='unsafe')))
         output_format = "CSF"
         output_name = "X"
+    
+    elif "concat_matmul.gv" in app_name:
+        # matrix b is completely dense
```

Comment:
remove comment that is not true for this test case

Reply:
done